File: problem-generator/4/4.js
console.log(15)
for (var i = 0; i < 15; i++) {
	var k = Math.floor(Math.random() * 20) + 1
  console.log(k)

	var correct = Math.floor(Math.random() * 100)

	for (var j = 0; j < k; j++) {
		var flip = Math.floor(Math.random() * 1000) % 2
		var lie = Math.floor(Math.random() * 1000) % 6
		var r = Math.floor(Math.random() * 100)

		var op = '<'
		if (correct > r && !flip) op = '>'
		if (r == correct) op = '='

		console.log(op + ' ' + r.toString() + ' ' + (flip ? 'No' : 'Yes'))
	}

	// console.log('correct: ' + correct.toString())
}

File: problem-generator/4/4.c
#include <stdio.h>

int main(void) {
	int n,i,j,t,k;
	scanf("%d",&k);
	while(k>0)
	{
		scanf("%d",&n);
		int fg=0,j=n,t=1;
		while(n>0)
		{
			i=n%10;
			if(i>=1)
			{
				fg=1;
				break;
			}
			n/=10;
			t*=10;
		}
		if(j==t)
		{
			fg=0;
		}
		if(fg)
			printf("YES\n");
		else
			printf("NO\n");
		k--;
	}
	return 0;
}
